code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -2937,3 +2956,201 @@
 	glass_icon_state = "berryjuice"
 	glass_name = "strawberry juice"
 	glass_desc = "Sweet and sugary, but also very refreshing!"
+
+//SoJ based drinks
+
+
+/datum/reagent/ethanol/southern_saki
+	name = "Southern Sake"
+	id = "southern_saki"
+	description = "Best served warm and in a shallow bowl for lapping."
+	taste_description = "unbearable bitterness"
+	color = "#F9F9F9" // rgb(98, 98, 98)
+	strength = 50 //rice wine stronk
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "sake"
+	glass_name = "Southern Sake"
+	glass_desc = "Best served warm and in a shallow bowl for lapping."
+
+/datum/reagent/ethanol/nothern_mead
+	name = "Nothern Mead"
+	id = "nothern_mead"
+	description = "Said to make even the strongest of accents all the more unintelligible."
+	taste_description = "robust hops and carbon"
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 140 //stronk
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "tankardlidfull"
+	glass_name = "Nothern Mead"
+	glass_desc = "Said to make even the strongest of accents all the more unintelligible."
+
+/datum/reagent/ethanol/life_line
+	name = "Life Line"
+	id = "life_line"
+	description = "A perfect drink for mass casualties."
+	taste_description = "liquid painkillers"
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 35 //weak painkiller
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "lifeline"
+	glass_name = "Life Line"
+	glass_desc = "A perfect drink for mass casualties."
+
+/datum/reagent/ethanol/posicell
+	name = "Posicell"
+	id = "posicell"
+	description = "The only drink that is shockingly annonying to make in and out of house."
+	taste_description = "rouge hardware and annoyed labcoats"
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 60
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "posicell"
+	glass_name = "Posicell"
+	glass_desc = "The only drink that is shockingly annonying to make in and out of house."
+
+/datum/reagent/ethanol/nickle_n_dime
+	name = "Nickle And Dime"
+	id = "nickle_n_dime"
+	description = "For those with a rich taste."
+	taste_description = "Nothing." //Nothing
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 120
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "nickleanddime"
+	glass_name = "Nickle And Dime"
+	glass_desc = "For those with a rich taste."
+
+/datum/reagent/ethanol/arcscrewdriver
+	name = "Arc screwdriver"
+	id = "arcscrewdriver"
+	description = "For one a regular one doesn't cut it."
+	taste_description = "electrooptic discussion"
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 75
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "arcscrewdriver"
+	glass_name = "Arc screwdriver"
+	glass_desc = "For one a regular one doesn't cut it."
+
+/datum/reagent/ethanol/whiskey_rocks
+	name = "Whiskey Rocks"
+	id = "whiskey_rocks"
+	description = "Whiske but for people that like things cold and waterd down."
```

Comment:
```suggestion
	description = "Whiskey but for people that like things cold and watered down."
```

Reply:
Should...should also be called "Whiskey IN the rocks"....unless these are literally solid rocks made out of whiskey...? 😂